plutus-metatheory/src/Algorithmic/Reduction.lagda

```diff
@@ -93,27 +102,17 @@ data _—→V_ : {A : ∅ ⊢Nf⋆ *} → (∅ ⊢ A) → (∅ ⊢ A) → Set wh
     → M —→V M'
     → wrap A B M —→V wrap A B M'
 
-  β-sbuiltin : ∀{A B}
+  β-sbuiltin : ∀{A B}{tn}
```

Comment:
I can't remember why this was called `sbuiltin` now, I think several versions coexisted for a while, the `s` can probably go.

Reply:
I've renamed it